Source/Core/Core/PowerPC/JitArm64/JitArm64_FloatingPoint.cpp

```diff
@@ -46,8 +46,12 @@ void JitArm64::fp_arith(UGeckoInstruction inst)
     VA = reg_encoder(fpr.R(a, type));
     if (use_b)
       VB = reg_encoder(fpr.R(b, type));
+    else
+      VB = {};
```

Comment:
assuming these changes quiet uninitialized variable warning. consider just initializing in ctor / at decl instead.